.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -1,9 +1,6 @@
 name: rawls-run-azure-e2e-tests
 
 on:
-  schedule:
```

Comment:
Why are we removing this?

Reply:
To ensure this doesn't get accidentally triggered on schedule during POC and testing. We'll reinstate it once this approach has been thoroughly vetted.